data/scenarios/Challenges/_dna/lab.sw

```diff
@@ -145,24 +144,19 @@ def makeDnaStrand = \receptacleLoc.
 
 def waitForCloneableOrganism =
 
-    waypointQuery <- waypoint "receiver" 0;
-    let receptacleLoc = snd waypointQuery in
+    let receptacleLoc = waypoint "receiver" 0 in
     organism <- instant (
         teleport self receptacleLoc;
 
         waitUntilOccupied;
 
-        thingHere <- scan down;
-        case thingHere (\x. return $ inL x) (\item.
-            isOrganism <- hastag item "organism";
-            return $ inR item;
-        );
+        scan down;
```

Comment:
@kostmo the existing code called `hastag item "organism"` but then ignored the result.  Was that a mistake?

Reply:
Oh, it probably is a mistake.  It should return `inL` if it is not an organism.